Rework `Beat Snap Divisor` and add `Beat Snap` (#5613)
ppy/osu-wiki: wiki/Beatmap_Editor/Beat_Snap_Divisor/en.md

# Beat Snap Divisor

![Beat Snap Divisor in the editor](img/beat_snap_divisor.png "Beat Snap Divisor in the editor")

> The **Beat Snap Divisor** is an editor setting that determines the editor's behaviour in quantising [hit objects](/wiki/Hit_Object) based on the [Beat Snap](/wiki/Beatmap_Editor/Beat_Snap) principles. It can be found in the upper-right portion of the editor screen, where it corresponds to the granularity of the objects on the [editor timeline](/wiki/Beatmap_Editor/Timelines); that is, a denser Beat Snap Divisor setting allows for more notes to be placed in the same measure and vice versa. 

Reviewer: ```suggestion
The **Beat Snap Divisor** is an editor setting that determines the editor's behaviour in quantising [hit objects](/wiki/Hit_object) based on the [Beat Snap](/wiki/Beatmap_Editor/Beat_Snap) principles. It can be found in the upper-right portion of the screen, where it corresponds to the granularity of the objects on the [timeline](/wiki/Beatmap_Editor/Timelines); that is, a denser Beat Snap Divisor setting allows for more notes to be placed in the same measure of time and vice versa. 
```

> The osu! editor supports eleven different Beat Snap Divisor settings as follows:

Reviewer: ```suggestion
The beatmap editor supports eleven different Beat Snap Divisor settings as follows:
```

This snap divisor is extremely uncommon out there and should be used with extreme care; in case you find yourself resorting to 1/5 snap divisor out of the blue, it is recommended for you to seek further advice from experienced mappers.

## 1/6 snap divisor

![](/wiki/shared/BSD_1_6.jpg "1/6 snap divisor in the osu!/osu!taiko/osu!catch editor")

![](/wiki/shared/1_6_m.jpg "1/6 snap divisor in the osu!mania editor")

The 1/6 snap divisor allows for hit objects to be placed on *double triplets*, which is the sixth fraction of a beat and its multiples (e.g. 1/6 and 5/6). It is represented by purple-coloured ticks on the editor timeline.

Although relatively rare, there are some songs and music genres (e.g. swing jazz) that primarily utilise this snap divisor throughout.

## 1/7 snap divisor

![](img/1_7_snap_divisor.png "1/7 snap divisor in the osu!/osu!taiko/osu!catch editor")

![](img/1_7_snap_divisor_m.png "1/7 snap divisor in the osu!mania editor")

The 1/7 snap divisor allows for hit objects to be placed on the seventh fraction of a beat and its multiples (e.g. 1/7, 2/7, 3/7, and so on). It is represented by yellow-coloured ticks on the editor timeline.

> This snap divisor is extremely uncommon out there and should be used with extreme care; in case you find yourself resorting to 1/7 snap divisor out of the blue, it is recommended for you to seek further advice from experienced mappers.

Reviewer: ```suggestion
This snap divisor is extremely uncommon and should be used with care. If you find yourself resorting to the 1/7 snap divisor out of the blue, it is recommended that you seek further advice from experienced mappers.
```

> ![1/7 snap divisor in the osu!mania editor](/wiki/shared/1_7_m.jpg "1/7 snap divisor in the osu!mania editor")

Reviewer: ```suggestion
![1/7 snap divisor in the osu!mania editor](/wiki/shared/1_7_m.png "1/7 snap divisor in the osu!mania editor")
```

## 1/8 snap divisor

![](/wiki/shared/BSD_1_8.jpg "1/8 snap divisor in the osu!/osu!taiko/osu!catch editor")

![](/wiki/shared/1_8_m.jpg "1/8 snap divisor in the osu!mania editor")

The 1/8 snap divisor allows for hit objects to be placed on the eighth fraction of a beat and its multiples (e.g. 1/8, 3/8, 5/8, and 7/8). It is represented by yellow-coloured ticks on the editor timeline.

## 1/9 snap divisor

![](img/1_9_snap_divisor.png "1/9 snap divisor in the osu!/osu!taiko/osu!catch editor")

![](img/1_9_snap_divisor_m.png "1/9 snap divisor in the osu!mania editor")

The 1/9 snap divisor allows for hit objects to be placed on the ninth fraction of a beat and its multiples (e.g. 1/9, 2/9, 3/9, and so on). It is represented by yellow-coloured ticks on the editor timeline.

> This snap divisor is extremely uncommon out there and should be used with extreme care; in case you find yourself resorting to 1/9 snap divisor out of the blue, it is recommended for you to seek further advice from experienced mappers.

Reviewer: ```suggestion
This snap divisor is extremely uncommon and should be used with care. If you find yourself resorting to the 1/9 snap divisor out of the blue, it is recommended that you seek further advice from experienced mappers.
```

> ![1/9 snap divisor in the osu!mania editor](/wiki/shared/1_9_m.jpg "1/9 snap divisor in the osu!mania editor")

Reviewer: ```suggestion
![1/9 snap divisor in the osu!mania editor](/wiki/shared/1_9_m.png "1/9 snap divisor in the osu!mania editor")
```

## 1/12 snap divisor

![](img/1_12_snap_divisor.png "1/12 snap divisor in the osu!/osu!taiko/osu!catch editor")

![](/wiki/shared/1_12_m.jpg "1/12 snap divisor in the osu!mania editor")

The 1/12 snap divisor allows for hit objects to be placed on the twelfth fraction of a beat and its multiples (e.g. 1/12, 5/12, 7/12, and so on). It is represented by grey-coloured ticks on the editor timeline.

> This snap divisor is extremely uncommon out there and should be used with extreme care; in case you find yourself resorting to 1/12 snap divisor out of the blue, it is recommended for you to seek further advice from experienced mappers.

Reviewer: ```suggestion
This snap divisor is extremely uncommon and should be used with care. If you find yourself resorting to the 1/12 snap divisor out of the blue, it is recommended that you seek further advice from experienced mappers.
```

## 1/16 snap divisor

![](img/1_16_snap_divisor.png "1/16 snap divisor in the osu!/osu!taiko/osu!catch editor")

![](/wiki/shared/1_16_m.jpg "1/16 snap divisor in the osu!mania editor")

The 1/16 snap divisor allows for hit objects to be placed on the sixteenth fraction of a beat and its multiples (e.g. 1/16, 3/16, 5/16, and so on). It is represented by grey-coloured ticks on the editor timeline.

> This snap divisor is extremely uncommon out there and should be used with extreme care; in case you find yourself resorting to 1/16 snap divisor out of the blue, it is recommended for you to seek further advice from experienced mappers.

Reviewer: ```suggestion
This snap divisor is extremely uncommon and should be used with care. If you find yourself resorting to the 1/16 snap divisor out of the blue, it is recommended that you seek further advice from experienced mappers.
```
